plugins/reports/api/reports.js

```diff
@@ -459,6 +503,15 @@ var metrics = {
                                     else {
                                         props["reports.report"] = localize.format(props["reports.report"], versionInfo.title);
                                         props["reports.your"] = localize.format(props["reports.your"], props["reports." + report.frequency], report.date);
+                                        props["reports.sent-by"] = localize.format(props["reports.sent-by"]);
+                                        props["reports.view-in-browser"] = localize.format(props["reports.view-in-browser"]);
+                                        props["reports.get-help"] = localize.format(props["reports.get-help"]);
+                                        const metricPropsString = {};
+                                        for (let k in metricProps) {
+                                            metricPropsString[k] = metricProps[k].map((item) => {
```

Comment:
Codacy found an issue: [Functions declared within loops referencing an outer scoped variable may lead to confusing semantics. (props)](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=6363153)